pkg/access_token/access_token_memory.go

```diff
@@ -0,0 +1,41 @@
+package access_token
+
+import (
+	"fmt"
+)
+
+type AccessTokenMemory struct {
+	Data []byte
+}
+
+func NewAccessTokenMemory() (token *AccessTokenMemory, err error) {
+	return &AccessTokenMemory{
+		Data: nil,
+	}, nil
+}
+
+func (token AccessTokenMemory) Read() (Data []byte, err error) {
```

Comment:
This is happening in a few places. The named returns aren't being used so they might as-well not be named. Unused named returns
1. Unnecessarily assign variables (I think but maybe the compiler just optimises them out) 
2. Can be misleading since the reader might be expecting to follow those named returns to understand what gets returned